### PR TITLE
dev-libs/amdgpu-pro-opencl: version bump 17.50-552542

### PR DESCRIPTION
**Pull request by brovko**
Bump 17.50 to patch 552542
Package-Manager: Portage-2.3.36, Repoman-2.3.9

### CODE REVIEW

**Comment by brovko** [comment]
Have found that on AMD site there are updated drivers for 17.50.
Should https://github.com/gentoo/gentoo/pull/9137 be adopted to support multilib (see https://github.com/gentoo/gentoo/pull/9113)?

File: dev-libs/amdgpu-pro-opencl/amdgpu-pro-opencl-17.50.552542.ebuild [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+LICENSE="AMD-GPU-PRO-EULA"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RESTRICT="mirror fetch strip"
+
+DEPEND="dev-util/patchelf"
+RDEPEND="dev-libs/ocl-icd"
+
+QA_PREBUILT="/opt/${SUPER_PN}/lib*/*"
```
Comment (gyakovlev) [inline dev-libs/amdgpu-pro-opencl/amdgpu-pro-opencl-17.50.552542.ebuild]: I think it should be `/opt/amdgpu` in `QA_PREBUILT`, not `${SUPER_PN}`
you are installing `into "/opt/amdgpu"` below.

**Comment by gyakovlev** [comment]
https://github.com/gentoo/gentoo/pull/9113 is useless without proper multilib in ocl-icd https://github.com/gentoo/gentoo/pull/9111

**Comment by candrews** [comment]
The latest version in portage is 18.20.606296 - why are we adding earlier versions?

**Comment by mkszuba** [comment]
My point exactly. In fact, I have been planning to remove 17.50 from the tree altogether by the end of this week.

**Comment by mkszuba** [comment]
Never mind, have just read https://bugs.gentoo.org/659676 which suggests version 18 doesn't always work right. Bumped 17.50.

**Pull request closed without merging**